packages/loot-core/src/types/models/simplefin.ts

```diff
@@ -15,7 +15,7 @@ export type SimpleFinAccount = {
 };
 
 export interface SimpleFinBatchSyncResponse {
-  [accountId: AccountEntity['account_id']]: BankSyncResponse;
+  [accountId: AccountEntity['id']]: BankSyncResponse;
```

Comment:
This was correct as account_id, was it giving errors?

<img width="328" alt="image" src="https://github.com/user-attachments/assets/e76eb3cb-76df-4597-a8a7-99393302583f" />
<img width="372" alt="image" src="https://github.com/user-attachments/assets/85511d80-8fa1-48c4-8878-e94d83be723e" />

Reply:
Oh hmm, it's missing from `AccountEntity` https://github.com/actualbudget/actual/blob/94600731d99e86fe89689eb776a103bd9556ae74/packages/loot-core/src/types/models/account.d.ts#L2. I guess the right thing to do is take it from `_SyncFields` instead